### DISCUSSION TITLE
RenderToString: How can I wrap long equation to new line?

### DISCUSSION THREAD

**Discussion by sahilsakhiya11** [KaTeX/KaTeX#3851] [Q&A]
**Describe the bug:**
When you have long equation than you div, equation goes out of the div. How can I start from new line if My equation doesn't fit to div. 

I am using renderToString method to generate html from latex and rendering it in div as <span>{mathFormulaGeneratedByRenderoStirng}<span>

**Screenshots:**
<img width="1344" alt="image" src="https://github.com/KaTeX/KaTeX/assets/37842013/ce8ce4ba-b438-48af-a291-1a9efd61bfbe">

**Comment by edemaine** [comment]
In display mode, there is no automatic line breaking (as in LaTeX). But if you use non-display mode, there will be automatic line breaking at the top level after operators like `+`.

See also https://github.com/KaTeX/KaTeX/issues/1023#issuecomment-1329853052 for a potential workaround for display mode, though I haven't tested it.
